vllm/datadog_checks/vllm/config_models/instance.py

```diff
@@ -0,0 +1,171 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
```

Comment:
hehe maybe codemodel-generator needs an update

Reply:
ah, I thought so too, but I have everything pointing to "latest" locally and I'm not getting any errors. I just pulled in latest master (should include the pydantic and co bump), will try again locally